x-pack/plugins/security_solution/public/common/lib/cell_actions/field_value.tsx

```diff
@@ -26,27 +27,30 @@ const useFormattedFieldProps = ({
   columnId,
   header,
   data,
+  unifiedDataTableRow,
 }: {
   rowIndex: number;
   data: TimelineNonEcsData[][];
   ecsData: Ecs[];
   header?: ColumnHeaderOptions;
   columnId: string;
   pageSize: number;
+  unifiedDataTableRow?: DataTableRecord & TimelineItem;
 }) => {
   const pageRowIndex = getPageRowIndex(rowIndex, pageSize);
-  const ecs = ecsData[pageRowIndex];
-  const link = getLinkColumnDefinition(columnId, header?.type, header?.linkField);
-  const linkField = header?.linkField ? header?.linkField : link?.linkField;
-  const linkValues = header && getOr([], linkField ?? '', ecs);
-  const eventId = (header && get('_id' ?? '', ecs)) || '';
+
   const rowData = useMemo(() => {
     return {
-      data: data[pageRowIndex],
+      data: unifiedDataTableRow ? unifiedDataTableRow.data : data[pageRowIndex],
       fieldName: columnId,
     };
-  }, [pageRowIndex, columnId, data]);
+  }, [pageRowIndex, columnId, data, unifiedDataTableRow]);
+  const ecs = unifiedDataTableRow ? unifiedDataTableRow.ecs : ecsData[pageRowIndex];
 
+  const link = getLinkColumnDefinition(columnId, header?.type, header?.linkField);
+  const linkField = header?.linkField ? header?.linkField : link?.linkField;
+  const linkValues = header && getOr([], linkField ?? '', ecs);
+  const eventId = (header && get('_id' ?? '', ecs)) || '';
```

Comment:
Reminder : Why are all these new definitions needed?

Reply:
🤷🏾‍♂️

Reply:
@michaelolo24 , sorry these comments are just for myself.. will remove it before I mark PR ready for review.

Reply:
These definitions are needed for custom rendering to cells in the table.